scalability_and_performance/ztp-deploying-disconnected.adoc

```diff
@@ -51,98 +59,166 @@ include::modules/ztp-acm-adding-images-to-mirror-registry.adoc[leveloffset=+3]
 
-include::modules/ztp-prerequisites-for-deploying-the-ztp-pipeline.adoc[leveloffset=+2]
+include::modules/ztp-checking-the-managed-cluster-status.adoc[leveloffset=+2]
 
-include::modules/ztp-installing-the-gitops-ztp-pipeline.adoc[leveloffset=+2]
+include::modules/ztp-configuring-the-cluster-for-a-disconnected-environment.adoc[leveloffset=+2]
 
-include::modules/ztp-preparing-the-ztp-git-repository.adoc[leveloffset=+3]
+include::modules/ztp-configuring-ipv6.adoc[leveloffset=+2]
 
-include::modules/ztp-preparing-the-hub-cluster-for-ztp.adoc[leveloffset=+3]
+include::modules/ztp-troubleshooting-the-managed-cluster.adoc[leveloffset=+2]
 
-include::modules/ztp-creating-the-site-secrets.adoc[leveloffset=+2]
+include::modules/ztp-applying-source-custom-resource-policies.adoc[leveloffset=+2]
 
-include::modules/ztp-creating-the-siteconfig-custom-resources.adoc[leveloffset=+2]
+include::modules/ztp-things-to-consider-when-creating-custom-resource-policies.adoc[leveloffset=+2]
 
-include::modules/ztp-creating-the-policygentemplates.adoc[leveloffset=+2]
+// TALO
 
-include::modules/ztp-checking-the-installation-status.adoc[leveloffset=+2]
+include::modules/cnf-topology-aware-lifecycle-operator.adoc[leveloffset=+1]
 
-include::modules/ztp-site-cleanup.adoc[leveloffset=+2]
+.Additional resources
 
-include::modules/ztp-removing-the-argocd-pipeline.adoc[leveloffset=+3]
+* For more information about the Topology Aware Lifecycle Operator, see xref:../scalability_and_performance/cnf-talo-for-cluster-upgrades.adoc#cnf-about-topology-aware-lifecycle-operator-config_cnf-topology-aware-lifecycle-operator[About the Topology Aware Lifecycle Operator].
 
-include::modules/ztp-troubleshooting-gitops-ztp.adoc[leveloffset=+1]
+* For more information about updating clusters in a disconnected environment, see xref:../updating/updating-restricted-network-cluster.adoc#update-service-overview_updating-restricted-network-cluster[Updating a restricted network cluster using the OpenShift Update Service].
 
-include::modules/ztp-validating-the-generation-of-installation-crs.adoc[leveloffset=+2]
+include::modules/cnf-topology-aware-lifecycle-operator-create-cgu-cr.adoc[leveloffset=+2]
 
-include::modules/ztp-validating-the-generation-of-policy-crs.adoc[leveloffset=+2]
+include::modules/cnf-topology-aware-lifecycle-operator-apply-policies.adoc[leveloffset=+2]
+
+include::modules/cnf-topology-aware-lifecycle-operator-precache-feature.adoc[leveloffset=+2]
+
+include::modules/cnf-topology-aware-lifecycle-operator-preparing-for-updates.adoc[leveloffset=+2]
+
+include::modules/cnf-topology-aware-lifecycle-operator-platform-update.adoc[leveloffset=+2]
+
+include::modules/cnf-topology-aware-lifecycle-operator-operator-update.adoc[leveloffset=+2]
+
+include::modules/cnf-topology-aware-lifecycle-operator-operator-and-platform-update.adoc[leveloffset=+2]
+
+include::modules/cnf-topology-aware-lifecycle-operator-troubleshooting.adoc[leveloffset=+2]
```

Comment:
This troubleshooting section seems general purpose for any use of TALO. It seems like it would be a better fit in the TALO specific documentation rather than under the ZTP doc.

Reply:
FYI @amolnar-rh